COMPILING.md

```diff
@@ -0,0 +1,161 @@
+Jamulus can be compiled for Linux, Windows and macOS as follows. Alernatively, you may wish to use one of the contributed [installation scripts](https://github.com/jamulussoftware/installscripts).
+
+---
+
+
+## Download sources
+
+For .tar.gz [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.tar.gz) to download the latest release
+
+For .zip [use this link](https://github.com/jamulussoftware/jamulus/archive/master.zip)
+
+## Linux
+
+#### Install dependencies
+
+On Ubuntu-based distributions 18.04+, Debian 9+ or 10 and Raspberry Pi Raspbian Buster release or later:
+
+* Build-essential
+* Qt5-qmake
+* Qtdeclarative5-dev
+* Qt5-default
+* Qttools5-dev-tools
+* Libjack-jackd2-dev
+
+#### On Fedora
+
+* Qt5-qtdeclarative-devel
+* Jack-audio-connection-kit-dbus
+* libQt5Concurrent5
+* Jack-audio-connection-kit-devel
+
+#### For all desktop distributions
+
+[QjackCtl](https://qjackctl.sourceforge.io/) is optional, but recommended to configure JACK.
+
+
+### Standard desktop build
+
+~~~
+qmake Jamulus.pro
+make clean
+make
+sudo make install
+~~~
+
+This puts the Jamulus binary into `/usr/local/bin`.
+
+#### Notes
+
+* The "make clean" command is essential to remove the automatically generated Qt files which are present in the .tar.gz file and may not match the Qt version you are using.
+
+* To use an external shared OPUS library instead of the built-in one use qmake `"CONFIG+=opus_shared_lib" Jamulus.pro`.
+
+* To use this file configure the software with `qmake "CONFIG+=noupcasename" Jamulus.pro` to make sure the output target name of this software is **j**amulus instead of **J**amulus.
+
+* Users of Raspberry Pi: You may want to compile the client on another machine and run the binary on the Raspberry Pi. In which case the only libraries you need to run it are those for a [headless server](Server-Linux#running-a-headless-server) build, but _with_ the JACK sound packages. In particular, have a look at the footnote for the headless build.
```

Comment:
Hrm, this sounds a bit like "ok, I'm going to compile in my workstation and run the result on my Raspberry", which is not going to work due to the architecture mismatch. I don't think we provide cross-compiling instructions either.

Reply:
Yes. That was initially intended because of the lack of .deb packages.

Reply:
Yes, I remember when this got written and it confused me I have to say. My understanding is that it would work if the distro (eg Ubuntu) was the same one each and all you wanted to do was save a few Kb and some build time on the Pi? Maybe @ann0see can clarify.

Reply:
>  all you wanted to do was save a few Kb and some build time on the Pi?

The main issues were:

1. No .deb files for Linux ARM (this issue still exists)
2. No headless .deb files for a server --> I had to clutter it with a GUI to just produce the headless build.

Reply:
> No .deb files for Linux ARM (this issue still exists)

Yes. I'll open an issue for that.

> Therefore I compiled the headless binary on my workstation and copied the binary to the server.

From your x86_64 workstation to a Raspberry server? I'm surprised that this can work?

Reply:
No. From my x86_64 PC to my x86_64 server. These were two different use cases.

Reply:
Ok, but then this ended up in a strange way in the docs. I read them as if it were possible to compile on my workstation and use the binary on my Raspberry. I think we should change or remove that.

Reply:
Can somebody verify that the notes for headless build are correct? It was referring to a non-existent "footnote" before which I think is [this in the original](https://jamulus.io/wiki/Compiling#the-headless-build-flag), but we agreed to remove some of it?

Reply:
> Can somebody verify that the notes for headless build are correct?

Did that as part of my last review.